src/meta/processors/id/GetSegmentIdProcessor.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+
+#ifndef META_GETSEGMENTIDPROCESSOR_H_
+#define META_GETSEGMENTIDPROCESSOR_H_
+
+#include "common/base/Base.h"
+#include "meta/processors/BaseProcessor.h"
+
+namespace nebula {
+namespace meta {
+class GetSegmentIdProcessor : public BaseProcessor<cpp2::GetSegmentIdResp> {
+ public:
+  static GetSegmentIdProcessor* instance(kvstore::KVStore* kvstore) {
+    return new GetSegmentIdProcessor(kvstore);
+  }
+
+  void process(const cpp2::GetSegmentIdReq& req);
```

Comment:
We also need tests for these new interface. Please refer to src/meta/tests/MetaClientTest.cpp